src/components/select/select.spec.js

```diff
@@ -258,6 +258,26 @@ describe('<md-select>', function() {
       }
     }));
 
+    it('displays selected-text when specified', inject(function($rootScope) {
+      var selectedText = "Hello World";
+      $rootScope.getSelectedText = function() { return selectedText; };
+
+      var select = setupSelect('ng-model="someVal", selected-text="getSelectedText()"', null, true).find('md-select');
```

Comment:
That works too, I figured if a test shows a function will be executed it should also cover the passed in string case, but not necessarily vice versa.